src/services/ui/src/components/Accordion/index.tsx

```diff
@@ -30,17 +27,17 @@ const AccordionTrigger = React.forwardRef<
   React.ElementRef<typeof AccordionPrimitive.Trigger>,
   AccordionTriggerProps
 >(({ className, children, ...props }, ref) => (
-  <AccordionPrimitive.Header className="flex">
+  <AccordionPrimitive.Header className="flex px-3 py-3 no-underline  text-lg font-bold">
     <AccordionPrimitive.Trigger
       ref={ref}
       className={cn(
-        "flex flex-1 items-center justify-between py-4 font-medium transition-all hover:underline [&[data-state=open]>svg]:rotate-180",
```

Comment:
i think the only thing in this file that needs to be changed is the `transition-all` bit. we can just take that out so it doesnt stack